.github/workflows/license-checker.yml

```diff
@@ -0,0 +1,18 @@
+name: license checker
+on: ['pull_request']
```

Comment:
only need to make a change if `package.json` is changed

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
I think you want it to be **package.json

Reply:
@rsmayda @htztomic for license validation I already have a check: which can be updated for allowed license values and it already checks for it when rush build is triggered for example [here](https://github.com/awslabs/solution-spark-on-aws/runs/7783880872?check_suite_focus=true#step:3:1774)

Reply:
@insignias this is checking if a project's dependencies have the correct license value

Reply:
ahh ok !